lib/Onyx.ts

```diff
@@ -346,11 +348,12 @@ function merge<TKey extends OnyxKey>(key: TKey, changes: OnyxMergeInput<TKey>):
                 return Promise.resolve();
             }
 
-            // For providers that can't handle delta changes, we need to merge the batched changes with the existing value beforehand.
```

Comment:
flagging this since I still have questions about `JSON_PATCH` before we remove it...